.github/workflows/publish-oss-for-cloud.yml

```diff
@@ -1,6 +1,6 @@
 name: Publish OSS Artifacts for Cloud
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ inputs.oss_ref || github.sha }}
```

Comment:
use either the tag or the sha?

Reply:
yup, the intent here was to limit concurrency based on the commit being built, so if the user specified a commit to build, use that, otherwise we are building the same commit as what we checked out, so use `github.sha`.